### PR TITLE
Include bench in analysis and compilation

### PR DESCRIPTION
**Pull request by nyurik**

### CODE REVIEW

File: .github/workflows/rust.yml [hunk 1]
```diff
@@ -17,6 +17,6 @@ jobs:
     steps:
     - uses: actions/checkout@v3
     - name: Build
-      run: cargo build --verbose
+      run: cargo build --workspace --verbose
     - name: Run tests
-      run: cargo test --verbose
+      run: cargo test --workspace --verbose
```
Comment (westnordost) [inline .github/workflows/rust.yml:22]: I didn't find anything in the docs about what this option is for

Comment (nyurik) [reply .github/workflows/rust.yml:22]: > $ cargo test --help
> ...
> --workspace               Test all packages in the workspace
> ...

Comment (nyurik) [reply .github/workflows/rust.yml:22]: if you have multiple packages (multiple Cargo.toml files), you should use `--workspace` when running compile/test/fmt/...

Comment (westnordost) [reply .github/workflows/rust.yml:22]: Ah I see, must have overlooked it. Thank you

**Pull request merged**